src/V3AstNodes.h

```diff
@@ -3567,12 +3584,17 @@ class AstAlwaysPostponed final : public AstNodeProcedure {
 
 class AstAlwaysPost final : public AstNodeProcedure {
     // Like always but post assignments for memory assignment IFs
+    AstNodeProcedure* m_upstreamp = nullptr;  // Upstream logic that triggers this always
```

Comment:
If you add references to nodes that are not op{1,2,3,4}p(), you need to override `const char* broken()` with the proper checks.

Reply:
And also typically have a cloneRelink().